docs/guides/primitives.mdx

```diff
@@ -17,45 +17,40 @@ In this context, a *primitive* is the smallest processing instruction, the simpl
-As we move toward larger QPUs (quantum processing units) and more complex workflows, the focus shifts from interacting with individual
-qubit signals to viewing quantum devices as systems that perform tasks we need.
+As the field moves toward larger quantum processing units (QPUs) and more complex workflows, the focus shifts from interacting with individual
+qubit signals to viewing quantum devices as systems that perform necessary tasks.
 
-The two most common tasks quantum computers are used for are sampling quantum states and calculating expectation values.
-These tasks motivated the design of the Qiskit primitives: Sampler and Estimator.
+The two most common tasks for quantum computers are sampling quantum states and calculating expectation values.
+These tasks motivated the design of the Qiskit primitives: **Estimator** and **Sampler**.
+
+- [Estimator](#estimator) computes expectation values of observables with respect to states prepared by quantum circuits.
+- [Sampler](#sampler) samples the output register from quantum circuit execution.
 
 In short, the computational model introduced by the Qiskit primitives moves quantum programming one step closer
 to where classical programming is today, where the focus is less on the hardware details and more on the results
 you are trying to achieve.
 
-## Implementation of Qiskit primitives
-
-The Qiskit primitives are defined by open-source primitive base-classes, from
-which different providers can derive their own Sampler and Estimator implementations. Among the implementations
-using Qiskit, you can find reference primitive implementations for local simulation in the `qiskit.primitives` module.
-Providers like Qiskit Runtime enable access to appropriate QPUs through native implementations of
-their own primitives.
+<Admonition type="note">
+The Qiskit SDK holds the definitions of the `Estimator` and `Sampler` primitives and acts  as the source-of-truth for this concept. It also contains reference implementations [(explained below)](./primitives#implementations) to assist providers of quantum compute resources in developing services that support Qiskit users.
 
-## Benefits of Qiskit primitives
+For users of IBM Quantum hardware, the [`qiskit-ibm-runtime`](./index#qiskit-runtime) package possesses an implementation of these primitives which allow you to execute quantum workloads on IBM Quantum&trade; hardware.
+</Admonition>
 
-For Qiskit users, primitives let you write quantum code for a specific QPU without having to explicitly
-manage every detail. Also, because of the additional layer of abstraction, you might be able to more easily
-access advanced hardware capabilities of a given provider. For example, with Qiskit Runtime primitives,
-you can leverage the latest advancements in error mitigation and suppression by toggling options such as `resilience_level`, rather than building your own implementation of these techniques.
+<span id="estimator"></span>
+## Estimator
```

Comment:
I think we should make it clearer that here we are talking about the runtime implementations of the primitives i know we mention it very briefly in the section above but its easy to miss. I think either update the heading or mention in the first paragraph of this section that this is demonstrating the runtime implementation. same deal for the sampler